### DISCUSSION TITLE
Recycle Bin and Storage Limit

### DISCUSSION THREAD

**Discussion by EdenSpire** [diced/zipline#362] [Ideas]
Hello,
I would like to give ideas about these two:

Recycle Bin:
To prevent a mistakenly click a delete button (I uploaded a big files and mistakenly click the delete icon (no warning), then the file vanish in a second(
To review a file before permanently delete.

Storage Limit (similar to #226 )
We have a upload size limit for users and admin.
It would be great if we can have max storage capacity for each user.

Great Software Btw!

**Comment by diced** [comment]
Recycle bin: I don't think I will add this in, though I think a warning for deleting files is a good thing to add in.
Storage Limit: planned in v4.
